docs/google/qubit-placement.ipynb

```diff
@@ -2,9 +2,8 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        !pip install --pre --quiet cirq
</pre><p>Let's add the note on top that this notebook depends on unreleased features and add the notebook to https://github.com/quantumlib/Cirq/blob/4ba978807b163c6d1ee51c1dea994c731d6400c5/dev_tools/notebooks/isolated_notebook_test.py#L43 so that we remember to remove the <code>--pre</code>  after next release. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/5321/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Good idea.  Done.